src/Illuminate/Foundation/Auth/Access/AuthorizesRequests.php

```diff
@@ -97,22 +112,60 @@ public function authorizeResource($model, $parameter = null, array $options = []
         }
     }
 
+    /**
+     * Set resource abilities.
+     *
+     * @param array $resourceAbilities
+     */
+    public function setResourceAbilities(array $resourceAbilities)
+    {
+        $this->resourceAbilities = $resourceAbilities;
+
+        return $this;
+    }
+
+    /**
+     * Use only the given abilities.
+     *
+     * @param string|array $abilities
+     * @return this
```

Comment:
```suggestion
     * @param  string|array  $abilities
     * @return $this
```

The same goes for other docblocks.

Reply:
Yes I mean `authorizeResource`. Sorry about the typo